src/main/java/Calculator.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Scanner;
+
+public class Calculator {
+
+    double sum; // общая сумма трат
+    double sumForGuest; // сумма для каждого гостя
+    String listProduct = "Добавленные товары:";
+    public void calculate(int peoples){
```

Comment:
```suggestion
    public void calculate(int people){
```
Слово и так во множественном числе)

Reply:
запарка)